contrib/utilities/jsontomarkdown.py

```diff
@@ -38,7 +38,8 @@ def handle_subsection(data, cur_path, output_file):
             print("(parameters:" + path_str + ")=", file=output_file)
             new_path = cur_path + [key]
             section_name = path_str.replace("_20", " ")
-            print("## **Parameters in section** " + section_name, file=output_file)
+            section_name = section_name.replace("/"," / ")
+            print("## **Subsection:** " + section_name, file=output_file)
```

Comment:
I think @tjhei removed the `** ... **` from the parameters already. I would favor it if we left the markup to sphinx's style sheets, rather than do it ourselves.